src/converter/r2t/EntityConverters.js

```diff
@@ -822,6 +831,21 @@ function _createTextElement($$, text, tagName, attrs) {
   }
 }
 
+function _createMultipleTextElements($$, text, tagName, attrs) {
+  if (text) {
+    const textItems = text.split('; ')
```

Comment:
If someone writes "foo;bar" without the space, it would not split it. So I guess here it would be better to split by ';' and then trim the whitespace for each item.